lib/ansible/plugins/filter/aws_kms.py

```diff
@@ -0,0 +1,52 @@
+""" Modoule to handle encrypting and decrypting of items with KMS """
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type  # pylint: disable=invalid-name
+import base64
+import botocore
```

Comment:
You'll need a try/catch around imports that are not from the Python standard library. See an example here:

https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/filter/json_query.py#L26

Reply:
Thanks @mattclay I've updated my changes accordingly :)